apps/webpack.js

```diff
@@ -44,6 +44,8 @@ var baseConfig = {
         include: [
           path.resolve(__dirname, 'static'),
           path.resolve(__dirname, 'src'),
+          path.resolve(__dirname, 'test'),
+          path.resolve(`${__dirname}/../dashboard/app/assets/`, 'images'),
```

Comment:
The main problem here is that presumably the files we are loading from dashboard are already being loaded elsewhere, by haml templates and what not. So people might end up downloading the same file twice, which is not super efficient. Solution would be to stick these files in apps and just not reference them from haml files at all. But that might be hard/time consuming to do?

Anyway, I think this is OK as is. If anyone deletes or moves that file, it will break the webpack build, so we don't have to worry about images going missing in production without notice.